apps/wallet-mobile/src/components/BlueCheckbox/BlueCheckbox.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import {ReactNode} from 'react'
+import {StyleSheet, TouchableOpacity, View} from 'react-native'
+import {ClipPath, Defs, G, Path, Rect, Svg} from 'react-native-svg'
+
+import {Spacer} from '../Spacer'
+
+export type BlueCheckboxProps = {
+  checked: boolean
```

Comment:
make booleans optional 
```suggestion
  checked?: boolean
```